Cargo.toml

```diff
@@ -146,13 +148,18 @@ required-features = ["electrum"]
 [[example]]
 name = "esplora_backend_synchronous"
 path = "examples/esplora_backend_synchronous.rs"
-required-features = ["use-esplora-ureq"]
+required-features = ["use-esplora-blocking"]
 
 [[example]]
 name = "esplora_backend_asynchronous"
 path = "examples/esplora_backend_asynchronous.rs"
 required-features = ["use-esplora-reqwest", "reqwest-default-tls", "async-interface"]
```

Comment:
If we are changing feature alias anyway, why not change this to `use-esplora-async` too.. Would be consistent with other examples..